_data/simple-icons.json

```diff
@@ -17044,6 +17044,12 @@
             "hex": "DA074A",
             "source": "https://commons.wikimedia.org/wiki/File:VOX_Logo_2013.svg"
         },
+        {
+            "title": "VRChat",
+            "hex": "000000",
+            "source": "https://drive.google.com/file/d/10BIsrBGt_4R7OQdYZIu6Nd5l1ogzOjVc/view?usp=drive_link",
+            "guidelines": "https://docs.google.com/presentation/d/e/2PACX-1vRlm7h3Vh5sjRVS478amvSywI1D7GWuMzv_kYkxOPW8dELnAVJq9Fbiqp_hnE1o2N-v2NyKOntdiaOz/pub?slide=id.g7ca235226a_0_13"
```

Comment:
As per https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines we should ideally link to a page where this Drive folder is *linked* to but not the Drive folder itself.

Reply:
That should take care of that